src/sass/patterns/_form-elements.scss

```diff
@@ -28,36 +28,25 @@
     }
   }
 
-  .slds-is-open {
-    > .slds-accordion__summary {
-      margin-bottom: $spacing-x-small;
-    }
-  }
-
   .grow-inner-item {
     .slds-accordion__summary-action > .slds-truncate {
       flex-grow: 1;
     }
   }
 
   .accordion-no-padding {
-    > .slds-accordion__list-item {
-      > .slds-is-open {
-        padding: 0;
-
-        > .slds-accordion__summary {
-          padding-top: $spacing-x-small;
-          padding-left: $spacing-medium;
-          padding-right: $spacing-medium;
-        }
+    .slds-is-open {
+      .slds-accordion__section {
+        padding-right: 0;
       }
     }
   }
 }
 
 .metecho-nested-checkboxes {
   // line up with above label; add grid + (faux checkbox + label margin)
-  padding-left: $square-icon-x-small-boundary + $spacing-large;
+  // twice left padding of accordion + icon and icon margin spacing
+  padding-left: 2.875rem;
```

Comment:
Curious why you changed this @stacyk . Was there a difference in sizing when using the tokens? I see the padding the same if I swap them out.

Reply:
@dvdherron Good question. Not much (it was 2.75 before) but I after some testing, I think the 2.75 was right but the thing I was missing was the third level of nesting wasn't lining up like the second level. I made some changes using SLDS custom properties as they are now available and not in Beta, and now the checkboxes line up with parent labels.